Sources/FirebaseFirestore/DocumentReference+Swift.swift

```diff
@@ -64,6 +64,35 @@ extension DocumentReference {
 
     return ListenerRegistration(boxed, instance)
   }
+
+  public func setData(_ data: [String: Any]) async throws -> Void {
+    try await setData(data, merge: false)
+  }
+
+  public func setData(_ data: [String: Any], merge: Bool) async throws -> Void {
```

Comment:
```suggestion
  private func setData(_ data: [String: Any], merge: Bool) async throws -> Void {
```

https://firebase.google.com/docs/reference/swift/firebasefirestore/api/reference/Classes/DocumentReference#setdata_:merge:

Seems to indicate that the public surface is synchronous.

Reply:
See above comment regarding API surfaces.